database/migrations/2025_02_17_115110_insert_blog_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('Blog', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
```

Comment:
It is good practice to leave `$table->timestamps();` as the last line of this method. The order of declarations has an effect on how the table appears in database management applications